cmake/deps.cmake

```diff
@@ -54,6 +54,10 @@ if(USE_SCIP)
   endif()
 endif()
 
+if(USE_GLPK)
```

Comment:
prefer to use like SCIP:
```cmake
if(USE_GLPK)
  if(NOT BUILD_GLPK)
    find_package(GLPK REQUIRED)
  endif()
endif()
```

Reply:
I couldn't figure out how to tell the super-build where GLPK is after building, so I made `FindGLPK.cmake` find it in the default install directories, depending on platform - so even if `BUILD_GLPK` is true, it still needs to find it

Reply:
That's the beauty of FetchContent since you'll have GLPK as sub-build you don't need to find it anymore.

note: currently in CMake you can't tell to find_package() command to perform a noop if already present
maybe in an other pass for all deps I should rewrite it as
```cmake
if(USE_GLPK)
  if(NOT BUILD_GLPK AND NOT TARGET GLPK::GLPK)
    find_package(GLPK REQUIRED)
  endif()
endif()
```
see: https://gitlab.kitware.com/cmake/cmake/-/issues/17735

Reply:
FYI I did the same in protobuf to make it super build friendly https://github.com/google/or-tools/blob/86d4c543f717fa8716a9d66c25143660668bf825/patches/protobuf-v3.18.0.patch#L32-L53 
so I sohuld make or-tools super build friendly too...